toolchain/semantics/semantics_handle_operator.cpp

```diff
@@ -61,7 +61,27 @@ auto SemanticsHandleInfixOperator(SemanticsContext& context,
 
 auto SemanticsHandlePostfixOperator(SemanticsContext& context,
                                     ParseTree::Node parse_node) -> bool {
-  return context.TODO(parse_node, "HandlePostfixOperator");
+  auto value_id = context.node_stack().PopExpression();
+
+  // Figure out the operator for the token.
+  auto token = context.parse_tree().node_token(parse_node);
+  switch (auto token_kind = context.tokens().GetKind(token)) {
+    case TokenKind::Star: {
+      auto inner_type_id = context.ExpressionAsType(parse_node, value_id);
+      context.AddNodeAndPush(
+          parse_node,
+          SemanticsNode::PointerType::Make(
+              parse_node,
+              context.CanonicalizeType(SemanticsNodeId::BuiltinTypeType),
+              inner_type_id));
+      break;
+    }
+
+    default:
+      return context.TODO(parse_node, llvm::formatv("Handle {0}", token_kind));
+  }
+
+  return true;
```

Comment:
```suggestion
      return true;
    }

    default:
      return context.TODO(parse_node, llvm::formatv("Handle {0}", token_kind));
  }
```

Just for your consideration, trying to simplify control flow.

Reply:
Sure, applied throughout this file for consistency.